lib/transforms/minifyCss.js

```diff
@@ -0,0 +1,33 @@
+const cssnano = require('cssnano');
+
+const defaultQueryObj = {type: 'Css', isLoaded: true};
+
+module.exports = queryObj => {
+    return async assetGraph => {
+        if (queryObj) {
+            queryObj = assetGraph.query.and(queryObj, defaultQueryObj);
+        } else {
+            queryObj = defaultQueryObj;
+        }
+        for (const cssAsset of assetGraph.findAssets(queryObj)) {
+            const result = await cssnano.process(cssAsset.parseTree, undefined, {
+                preset: [
+                    'default',
+                    {
+                        svgo: false,
+                        safe: true,
+                        normalizeUrl: false, // Buggy
+                        mergeLonghand: false,
```

Comment:
Longhand merging seems useful. Does enabling it cause trouble somehow?

Reply:
There used to be a bug that caused it to kick in even when all the longhand properties weren’t defined, so it didn’t preserve the semantics, yet was classified as a “safe” transform. It caused problems in a project of mine where some styling got mangled.

Sounds like [it’s fixed now](https://github.com/ben-eb/cssnano/issues/358#issuecomment-283696193). We can certainly try re-enabling it, but Ididn’t want to mix that into this pr.